Update documentation (#7)
serpapi/serpapi-ruby: README.md

[![serpapi-ruby](https://github.com/serpapi/serpapi-ruby/actions/workflows/ci.yml/badge.svg)](https://github.com/serpapi/serpapi-ruby/actions/workflows/ci.yml) [![Gem Version](https://badge.fury.io/rb/serpapi.svg)](https://badge.fury.io/rb/serpapi) 

Integrate search data into your AI workflow, RAG / fine-tuning, or Ruby application using this official wrapper for [SerpApi](https://serpapi.com). 

SerpApi supports Google, Google Maps, Google Shopping, Baidu, Yandex, Yahoo, eBay, App Stores, and [more](https://serpapi.com). 

> Fast query at scale a vast range of data, including web search results, flight schedules, stock market data, news headlines, and [more](https://serpapi.com). 

Reviewer: minor grammatical

## Search API advanced usage with Google search engine

This example dives into all the available parameters for the Google search engine.

> The set of parameters is extensive and depends on your chosen search engine.

Reviewer 2: ```suggestion
The list of parameters depends on the chosen search engine.
```
